### PR TITLE
[NUI] Add new rotation constructor

### PR DESCRIPTION
**Pull request by bshsqa**
- Adds new Rotation Constructor that uses two Vector3

Signed-off-by: Seungho Baek <sbsh.baek@samsung.com>

### CODE REVIEW

**Review by dongsug-song — APPROVED** [review]
looks good to me. 
but dali-csharp-binder patch should be merged first or integrated at the same time.

**Comment by bshsqa** [comment]
Thank you.
csharp binder patch is on testing in the build server.
After it is finished I will merge it first.

File: src/Tizen.NUI/src/public/Common/Rotation.cs [hunk 1]
```diff
@@ -48,6 +48,17 @@ public Rotation(Radian angle, Vector3 axis) : this(Interop.Rotation.NewRotation(
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// The constructor from two vector.
+        /// </summary>
+        /// <param name="v0">Thefirst vector.</param>
+        /// <param name="v1">The second vector.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Rotation(Vector3 v0, Vector3 v1) : this(Interop.Rotation.NewRotation2(Vector3.getCPtr(v0), Vector3.getCPtr(v1)), true)
```
Comment (myroot) [inline src/Tizen.NUI/src/public/Common/Rotation.cs:57]: Please give detail information about v0, v1

Comment (bshsqa) [reply src/Tizen.NUI/src/public/Common/Rotation.cs:57]: Thanks I added some more information about this constructor.

**Pull request merged**